examples/backtest.ipynb

```diff
@@ -23,12 +23,14 @@
     "\n",
```

Comment:
<p>You don't need to load the dataset the second time, backtest doesn't touch it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>